classes/Risk.php

```diff
@@ -63,16 +71,25 @@ public function getFields()
     public function getTranslationsFieldsChild()
     {
         $this->validateFieldsLang();
-        return $this->getTranslationsFields(array('name'));
+
+        return $this->getFieldsLang();
```

Comment:
@firstred I've no idea. This is the only class that has this feature...Waiting for another feedback ..